bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/RowFilterAdapter.java

```diff
@@ -37,16 +40,26 @@
   TypedFilterAdapter<org.apache.hadoop.hbase.filter.RowFilter> {
 
   @Override
-  public RowFilter adapt(FilterAdapterContext context, 
-      org.apache.hadoop.hbase.filter.RowFilter filter)
-      throws IOException {
+  public RowFilter adapt(FilterAdapterContext context,
+      org.apache.hadoop.hbase.filter.RowFilter filter) throws IOException {
+    if (filter.getOperator() != CompareFilter.CompareOp.EQUAL) {
+      throw new IllegalStateException(String.format("Cannot adapt operator %s", filter
+          .getOperator().getClass().getCanonicalName()));
```

Comment:
In practice that won't happen, because isFilterSupported() will catch that.  I'll check it here anyway.